Sources/NIOHTTPClient/HTTPHandler.swift

```diff
@@ -207,13 +224,15 @@ internal extension URL {
 
 public extension HTTPClient {
     final class Task<Response> {
+        let eventLoop: EventLoop
```

Comment:
I think this needs to be `public`. If I'm correct the tests are only compiling because you have `@testable` which users won't have...

Reply:
fixed, thanks!

Reply:
yeah, if we can it might be good to remove `@testable` from the test files... But I know it's not always that straightforward. Another option might be to have to test suites: One for the public API and one that does some extra internal stuff. But this is not super high priority ofc.